datalad/core/local/create.py

```diff
@@ -208,24 +203,10 @@ def __call__(
             force=False,
             description=None,
             dataset=None,
-            no_annex=_NoAnnexDefault,
             annex=True,
             fake_dates=False,
             cfg_proc=None
     ):
-        # TODO: The current release of datalad-metalad (v0.2.1) still uses
-        # no_annex in its tests. Remove this compatibility kludge once a
-        # release is made, which will include 16a170e (2020-09-08).
-        if no_annex is not _NoAnnexDefault:
-            # the two mirror options do not agree and the deprecated one is
-            # not at default value
-            warnings.warn("datalad-create's `no_annex` option is deprecated "
-                          "and will be removed in a future release, "
-                          "use the reversed-sign `annex` option instead.",
-                          DeprecationWarning)
```

Comment:
It's not particularly important, so I have no objections to removing this, but note that the description in the comment isn't stale: metalad still hasn't been released, so the current release still uses `no_annex` in its tests.

Reply:
oh, I missed that.  I guess I will just merge without this commit ? or may be we should just encourage datalad-metalad to release? ;) WDYT @christian-monch @mih

Reply:
TODO for the merge to add `#!/usr/bin/env python3` shebang in the light of #5436 (don't want to torture CI for no reason)

Reply:
> I guess I will just merge without this commit ?

All right, if you're okay with that, that'd be my preference.

> TODO for the merge to add `#!/usr/bin/env python3`

Sounds good.

Both done with the merge in 76d9c36eac.